llvm/docs/RISCVUsage.rst

```diff
@@ -96,6 +96,8 @@ on support follow.
      ``Svnapot``      Assembly Support
      ``Svpbmt``       Supported
      ``V``            Supported
+     ``Za128rs``      Supported
```

Comment:
I think these may warrant an explanatory note after the table.  See what we do for e.g. zicntr.  Something which says these are defined by the profile specification seems like a useful breadcrumb.